custom-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,421 @@
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          }
+        },
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "callbackURL": {
+                  "type": "string",
+                  "description": "callbackURL",
+                  "title": "CallbackURL",
+                  "x-ms-notification-url": false,
+                  "x-ms-visibility": "internal"
+                },
+                "entityName": {
+                  "type": "string",
+                  "description": "entityName",
+                  "title": "EntityName",
+                  "x-ms-visibility": "internal",
+                  "default": "Ticket"
+                },
+                "triggerGroup": {
+                  "type": "string",
+                  "description": "triggerGroup",
+                  "title": "TriggerGroup",
+                  "enum": [
+                    "Added",
+                    "Updated"
+                  ]
+                }
+              },
+              "required": [
+                "callbackURL",
+                "entityName",
+                "triggerGroup"
+              ]
+            }
+          }
+        ],
+        "summary": "Ticket",
+        "description": "This is triggered whenever a new or an update of ticket entity in the ticket management system.",
+        "operationId": "TicketAddedUpdated",
+        "consumes":[
+          "application/json"
+        ],
+        "produces":[
+          "application/json"
+        ]
+      }
+    },
+    "/cc/Triggers/RegisterWebhookFile": {
+      "x-ms-notification-content": {
+        "description": "FileResponse"
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          }
+        },
+        "summary": "File",
+        "description": "This is triggered whenever a new or an update of files entity in the ticket management system.",
+        "operationId": "FileAddedUpdated",
+        "consumes":[
+          "application/json"
+        ],
+        "produces":[
+          "application/json"
+        ],
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "callbackURL": {
+                  "type": "string",
+                  "description": "callbackURL",
+                  "title": "CallbackURL",
+                  "x-ms-notification-url": false,
+                  "x-ms-visibility": "internal"
+                },
+                "entityName": {
+                  "type": "string",
+                  "description": "entityName",
```

Comment:
@1MrSharma All the description values in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Reply:
Updated the changes required.